src/test/java/com/ays/admin_user/service/impl/AdminUserRegisterServiceImplTest.java

```diff
@@ -210,22 +168,17 @@ void givenExistingEmailFromAdminUserRegisterRequest_whenAdminUserExist_thenThrow
         Mockito.when(adminUserRegisterApplicationRepository.findById(Mockito.anyString()))
                 .thenReturn(Optional.ofNullable(mockAdminUserRegisterApplicationEntity));
 
-        Mockito.when(institutionRepository.existsById(Mockito.anyString()))
-                .thenReturn(true);
-
         Mockito.when(adminUserRepository.existsByEmail((Mockito.anyString())))
                 .thenReturn(true);
 
         // Then
         Assertions.assertThrows(
                 AysAdminUserAlreadyExistsByEmailException.class,
-                () -> adminUserRegisterService.register(mockAdminUserRegisterRequest)
+                () -> adminUserRegisterService.completeRegistration(applicationId, mockAdminUserRegisterApplicationCompleteRequest)
         );
 
         Mockito.verify(adminUserRegisterApplicationRepository, Mockito.times(1))
```

Comment:
Aşağıdaki gibi suggestion kısmını da ekleyebilir miyiz?
```suggestion
        // Verify
        Mockito.verify(adminUserRegisterApplicationRepository, Mockito.times(1))
```

Reply:
Ekledim.